confluence/confluence-v1.json

```diff
@@ -0,0 +1,131 @@
+{
+    "parserId": "{add parserid }",
+    "created": "2024-08-21T14:34:00.546Z",
+    "coreProperties": {},
+    "modified": "2024-08-21T20:00:35.287Z",
+    "id": "confluence",
+    "type": "lucidworks.rest",
+    "properties": {
+        "serviceURL": "https://{add confluence instance}",
+        "authenticationMode": {
+            "basicAuth": {
+                "password": "xXx-Redacted-xXx",
+                "user": "add email address here!!!"
+            }
+        },
+        "serviceEndpoints": [
+            {
+                "childrenRequests": [
+                    {
+                        "dataRequest": {
+                            "endpoint": "/wiki/rest/api/content/${LW_PARENT_DATA_KEY}/child/comment",
+                            "httpMethod": "GET",
+                            "queries": [
+                                {
+                                    "queryKey": "expand",
+                                    "queryValue": "body.storage"
+                                }
+                            ]
+                        },
+                        "childResponseMapping": {
+                            "childDataPath": "results",
+                            "parentIdKey": "id"
+                        }
+                    }
+                ],
+                "rootRequest": {
+                    "dataRequest": {
+                        "endpoint": "/wiki/rest/api/content",
+                        "pagination": {
+                            "paginationByBatchSize": {
+                                "batchSize": 1,
```

Comment:
Maybe it should be better to recommend a higher batchSize (5 or maybe the default 25?) , so less requests will be made. What do you think?